xbmc/video/VideoDatabase.cpp

```diff
@@ -394,12 +394,19 @@ void CVideoDatabase::CreateViews()
                                        "        episode.idShow=tvshow.idShow"
                                        "      LEFT JOIN files ON"
                                        "        files.idFile=episode.idFile "
-                                       "    GROUP BY tvshow.idShow");
+                                       "GROUP BY tvshow.idShow");
   m_pDS->exec(tvshowcounts);
 
   CLog::Log(LOGINFO, "create tvshow_view");
   std::string tvshowview = PrepareSQL("CREATE VIEW tvshow_view AS SELECT "
-                                     "  tvshow.*,"
+                                     "  tvshow.idShow AS idShow,"
+				     "  tvshow.c00 AS c00, tvshow.c01 AS c01, tvshow.c02 AS c02, tvshow.c03 AS c03,"
```

Comment:
Here also the column number constants should be used with PrepareSQL. I know it's tedious, but it's the only clean way with the strange design that we have. You can find the list of all constants and their values in `VideoDatabase.h`.

Reply:
Actually... I think you can leave `tvshow.*` here. `tvshow_view` should behave mostly like `tvshow`, so including all columns is natural and we do not have a `GROUP BY` so there is no need to spell them out explicitly. Sorry for the confusion.

Reply:
Ok, changed back tvshow_view to SELECT tvshow.* as there is no way I can safely understand which labels to use for the tvshow.c%02d columns....
Possibly you guys need to create a wiki to document the logic of this videodatabase.h and which set of labels correspond to which tables...

Reply:
You mean https://kodi.wiki/view/Databases/MyVideos
Database is gong to be redesigned anyway soon so if you are willing to help on that
https://github.com/xbmc/xbmc/tree/feature_odb

Reply:
Cool, many thanks. Let me know how I can help...

Reply:
@lobermann Feel free to ask if you have something concrete you need help with. I'm not really up-to-date on the DB rework state.